content/tutorials/style_inbo_bibliography/index.Rmd

```diff
@@ -26,27 +25,23 @@ All bibliography managers provide ways to automate the formatting of citations a
 Depending on the bibliographic manager software, the style formatting rules are stored in specific files.
 In this tutorial you will learn how to use the INBO citation style in EndNote, Mendeley and Zotero bibliography managers.
 
-EndNote uses a `*.ens` file to store the style formatting rules, whereas Mendeley and Zotero use the open source `*.csl` file (Citation Style Language)[https://citationstyles.org/].
+EndNote uses a `*.ens` file to store the style formatting rules, whereas Mendeley and Zotero use the open standard `*.csl` [Citation Style Language](https://citationstyles.org/).
+EndNote, Zotero and Mendeley have recommended ways of finding and installing these files and will always point to the latest version of the style files when properly installed.
+This is explained in [Use in bibliography managers].
 
-The latest official CSL file of the INBO citation style is available at the CSL style repository (also transferred automatically to the Zotero style repository).
-See (finding-and-installing-styles)[https://citationstyles.org/authors/#/finding-and-installing-styles] for more information.
+The latest official CSL file of the INBO citation style is available at the CSL style repository (also transferred automatically to the Zotero style repository) and can be found [here](https://github.com/citation-style-language/styles/blob/master/research-institute-for-nature-and-forest.csl).
```

Comment:
```suggestion
In case you want to manually download the latest official CSL file of the INBO citation style (not the recommended way; better use your bibliography manager): it is [available](https://github.com/citation-style-language/styles/blob/master/research-institute-for-nature-and-forest.csl) at the CSL style repository.
```